### DISCUSSION TITLE
CustomScriptExtension i copyindex()

### DISCUSSION THREAD

**Discussion by Maniek666** [AzureCommunityPL/group#90] [Pomoc] [answered]
Hej,

Mam pytanie, zbudowałem sobie ARMa, gdzie tworze sobie 2 VMki za pomocą copyindex(), żeby nie powielać. Teraz chciałbym, żeby podczas ich tworzenia uruchomić skrypt powershell'owy. Wszystko działa za pomocą CustomScriptExtension jeśli maszyny tworze bez copyindex. Niestety w składni CustomScriptExtension dostaje błąd, że nie mozna użyc copyindex.

Dzięki.

**Comment by MarczakIO** [comment]
Pewnie znajda się osoby chętne do pomocy, ale idąc za zasada z innych forów musisz dostarczyć tzw. minimalny reprodukowalny przykład. Podrzuć ARM który ktoś może poprawić lub zerknąć gdzie masz błąd.

**Comment by kamil-mrzyglod** [comment]
Takie moje wolne przemyślenie - jeśli to 2 VMki to ja osobiście bym się zastanowił jaką przewagę ma pętla ze swoją złożonością nad kopiuj-wklej w tym konkretnym wypadku ;)

Tak jak @MarczakIO napisał - wrzuć szablon albo snippeta bo nie bardzo wiadomo czy to błąd składniowy czy faktycznie problem z CSE.

**Comment by Maniek666** [comment, marked as answer]
Dzięki za komentarze, udało mi się ogarnąć temat. Jeśli ktoś byłby zainteresowany to mogę wrzucić część ARMa. Czasami nie wszystko piszą na tych stronach MS ;)

Dzięki!

**Reply by kamil-mrzyglod** [reply]
Wrzuć dla potomnych :)
